docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -38,6 +39,43 @@ function cgov_site_section_entity_bundle_field_info(EntityTypeInterface $entity_
   }
 }
 
+/**
+ * Implements term_cgov_site_sections_parent_validate(), validate default root as parent.
+ */
+function term_cgov_site_sections_parent_validate(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+   $langcode =  $form_state->getValue('langcode');
+   $tid =  $form_state->getValue('tid');
+   $vid = 'cgov_site_sections';
+    $terms =\Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree($vid);
+    $enParentTermCount = $esParentTermCount = 0;
+    foreach ($terms as $term) {
+      if($term->depth == 0 && $term->langcode == 'en' && $tid != $term->tid) {
+        $enParentTermCount++;
+      }
+      if($term->depth == 0 && $term->langcode == 'es' && $tid != $term->tid) {
+        $esParentTermCount++;
+      }
+    }
+
+    $entity = $form_state->getFormObject()->getEntity();
+    $parent =  $form_state->getValue('parent');
+   if(in_array(0, $parent)) {
+       if(($enParentTermCount >= 1 && $langcode[0]['value'] == 'en') || ($esParentTermCount >= 1 && $langcode[0]['value'] == 'es')) {
+          $form_state->setErrorByName('parent', t('The site section cannot be saved. Please set the parent site section under the Relations tab.'));
+       }
+
+}
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function cgov_site_section_form_alter(&$form, FormStateInterface &$form_state, $form_id) {
```

Comment:
I initially typed the following, but there may be a better way:
>There should be a `hook_entity_presave` implementation. `hook_form_alter` only covers using the UI. Using the Drupal API, I can create multiple taxonomy terms through the backend API.  We should not be able to do bad things through the backend. 

However, a quick bit of Googling to see how an exception in presave is handled, I see that you can add a Constraint and handle both UI and API.

See 
* https://drupal.stackexchange.com/questions/183685/how-do-i-add-a-custom-validation-handler-to-an-existing-form-field.
* https://www.drupal.org/docs/drupal-apis/entity-api/entity-validation-api/providing-a-custom-validation-constraint#add_constraint_other_entity_base_field

The key take away is that I should not be able to create multiple root site sections ever. Not through the UI, not through the backend.